libcontainer/configs/spec.go

```diff
@@ -32,6 +34,13 @@ func ParseSpec(s *specs.Spec) (*Config, error) {
 		}
 	}
 
+	// Setting default memory to pass to runnc as an argument.
+	if s.Linux.Resources.Memory.Limit != nil {
+		memory = (*s.Linux.Resources.Memory.Limit) / (1 << 20)
+	} else {
+		memory = 512
```

Comment:
Since there's multiple packages and right now it looks like this constant would touch `specs.go` in config package and `runnc_cont.go` in its main package do you have a suggestion where I should put it?